tests/bioentities.bats

```diff
@@ -209,6 +209,19 @@ setup() {
   [ "${status}" -eq 0 ]
 }
 
+@test "[bioentities] Check that optimisation worked" {
+  if [ -z ${SOLR_HOST+x} ]; then
+    skip "SOLR_HOST not defined, skipping load to Solr"
+  fi
+  COLLECTION=bioentities-v1
+
+  # result stores the number of deletedDocs for each core/shard of the collection
+  run curl "http://${SOLR_HOST}/solr/admin/collections?action=CLUSTERSTATUS&collection=${COLLECTION}" | jq '..|.replicas? | select( . != null ) | to_entries | .[] | .value | (.base_url|tostring)+"/admin/cores?action=STATUS&core="+(.core|tostring)' | xargs curl -s | jq '..|.deletedDocs? | select( . != null )' | uniq
```

Comment:
Shouldn’t this be `result=$(cmd)`?

Reply:
I think that bats needs at least one run statement. I noticed that apparently the run statement didn't like the pipings. Putting everything on a separate bash script and then run of that works.